manifests/artemis.pp

```diff
@@ -22,7 +24,7 @@
 
   file { "${candlepin::catalina_home}/conf/cert-users.properties":
```

Comment:
I guess this then needs something like `require => File[$candlepin::artemis_client_certificate]`